parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -388,6 +415,23 @@ pub mod pallet {
 		}
 
 		fn process_finalized_header(update: FinalizedHeaderUpdateOf<T>) -> DispatchResult {
+			let import_time = <LatestFinalizedHeaderImportTime<T>>::get();
+			let weak_subjectivity_period_check = import_time + 97200;
```

Comment:
seems better to move to a constant instead of hardcode

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/742/commits/fb7483fd3fefee6127f2e5e5a100c93a7ec9b3a4.